packages/mobile/src/fees/saga.ts

```diff
@@ -211,7 +211,7 @@ function* mapFeeInfoToUsdFee(feeInfo: FeeInfo) {
     tokensByCurrencySelector
   )
   const tokenInfo = tokensInfo[feeInfo.currency]
-  if (!tokenInfo) {
+  if (!tokenInfo?.usdPrice) {
     throw new Error(`No token info found for ${feeInfo.currency}`)
```

Comment:
nit: I'm not sure if this error.message is logged, but its a misleading after the change above (either tokenInfo or usdPrice can by falsey).  Suggestion:

```
`Missing tokenInfo or tokenInfo.usdPrice for ${feeInfo.currency}`
```

Reply:
Yes, to clarify this shouldn't happen since the feeCurrency should only be CELO, cUSD, cEUR or cREAL, all of which have usd price, but it's good to make the message clearer